packages/amplication-build-manager/src/build-runner/build-runner.service.spec.ts

```diff
@@ -672,7 +669,7 @@ describe("BuildRunnerService", () => {
           .spyOn(service, "getCodeGeneratorVersion")
           .mockResolvedValue(codeGeneratorVersion);
 
-        jest.spyOn(service, "copyFromJobToArtifact").mockResolvedValue(true);
+        jest.spyOn(service, "copyFromJobToArtifact");
```

Comment:
Not related to your change but we can make this test better by faking an exception here.

```suggestion
    jest
      .spyOn(service, "copyFromJobToArtifact")
      .mockRejectedValueOnce(new Error("failed to copy"));```